armadillo/src/main/resources/application.yml

```diff
@@ -7,6 +7,12 @@ rserve:
 datashield:
   # set this to 'true' if you want to accept permissions from oidc provider
   oidc-permission-enabled: false
+
+  # Uncomment this section to bootstrap the application with default values.
+  # bootstrap:
+    # oidc-admin-user: user@yourdomain.org
+    # default-project: your-project
```

Comment:
what is this for?

Reply:
`default-project` can be used during development. In the data folder we have the project `lifecycle` but every time you remove/reset the system folder with the metadata, you have to manually add lifecycle as a project again. Now you can just create your own `application-local.yml` (see readme), configure that project once and be done with it.

Reply:
that is great.